### PR TITLE
Throttle angular-drag-and-drop library 'dragover' placeholder re-draw to prevent performance hit

### PR DESCRIPTION
**Pull request by ghobs91**

### CODE REVIEW

File: angular-drag-and-drop-lists.js [hunk 1]
```diff
@@ -409,6 +417,10 @@
           if (!data) return stopDragover();
         }
 
+        if (typeof data === "object") {
```
Comment (vishnu-kulkarni) [inline angular-drag-and-drop-lists.js:425]: This change is new. Whats is this for?

File: angular-drag-and-drop-lists.js [hunk 2]
```diff
@@ -372,16 +381,15 @@
        * one child element per array element.
        */
       element.on('drop', function(event) {
+        // The default behavior in Firefox is to interpret the dropped element as URL and
+        // forward to it. We want to prevent that even if our drop is aborted.
+        event.preventDefault();
```
Comment (vishnu-kulkarni) [inline angular-drag-and-drop-lists.js]: Is there any specific reson to move this ?>

File: angular-drag-and-drop-lists.js (hunk 1 above)
Comment (ghobs91) [reply angular-drag-and-drop-lists.js:425]: Safety check to ensure an empty/null data doesn't cause an error. I observed it happening.

File: angular-drag-and-drop-lists.js (hunk 2 above)
Comment (ghobs91) [reply angular-drag-and-drop-lists.js]: Initially appeared to be necessary during testing, but doesn't look to be the case now. Moved it back.

**Pull request closed without merging**